packages/react-storage/src/components/StorageBrowser/providers/configuration/credentials/types.ts

```diff
@@ -13,9 +13,8 @@ import {
 export type RegisterAuthListener = (onStateChange: () => void) => void;
 
 export type GetCredentials = (input: {
-  bucket: string;
+  scope: string;
```

Comment:
IIRC, we want to keep it bucket and prefix since with permissions work we plan on spliting the scope type into bucket & prefix. cc: @calebpollman should this just retain this?

Reply:
nvm this is unrelated to that i think from looking down below.